src/Protocol/Palettes/Palette_1_14.h

```diff
@@ -1,12 +1,13 @@
 #pragma once
 
-#include "../../Registries/Items.h"
-#include "../../Registries/Statistics.h"
+#include "BlockState.h"
+#include "Registries/Items.h"
+#include "Registries/Statistics.h"
```

Comment:
Include paths should be relative.

Reply:
could you remind me why? the ../../../ looks unaesthetic.

Reply:
IDK, I think it was xoft's preference. Either way we shouldn't mix and match, lets stick to how the rest of the code does it.

Reply:
hehe, you are also invoking the principle of consistency I see ;P

Reply:
Until the issue you opened on this topic is resolved, this should be made relative again.